ArmPlatformPkg/Sec/Sec.c

```diff
@@ -80,6 +80,14 @@ STATIC CONST EFI_PEI_PPI_DESCRIPTOR  gCommonPpiTable[] = {
   }
 };
 
+// The Transfer List PPI is used to pass the Transfer List to the PEI Core
+// and goes at the end of the PPI list.
+STATIC EFI_PEI_PPI_DESCRIPTOR  mTransferListBase = {
+  (EFI_PEI_PPI_DESCRIPTOR_PPI | EFI_PEI_PPI_DESCRIPTOR_TERMINATE_LIST),
+  &gArmTransferListPpiGuid,
+  NULL // Set later by SecMain

```

Comment:
SEC and PEI do not permit writable global variables. You will need to allocate this PPI dynamically if you want to modify it at runtime. You may be able to steal some space from the base of the stack like the code below is already doing.